kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/templates/kroxylicious/KroxyliciousKafkaClusterRefTemplates.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.templates.kroxylicious;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaServiceBuilder;
+import io.kroxylicious.systemtests.Constants;
+
+public class KroxyliciousKafkaClusterRefTemplates {
```

Comment:
I'm not convinced that we need all the `*Template` classes, we we want the static signatures then I think we would be better collapsing them into a single Templates class with methods for each type.